pkgs/tools/package-management/disnix/default.nix

```diff
@@ -18,7 +18,7 @@ stdenv.mkDerivation {
   meta = {
```

Comment:
tiny nit: this and the dydisnix meta attrsets would be cleaner with `with stdenv.lib;` like you've done in libnixxml, but it shouldn't block merging this.